docs/content/users/quickstart.md

```diff
@@ -397,6 +397,37 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     ddev launch
     ```
 
+=== "Laravel Installer"
+
+    ```bash
+    mkdir my-laravel-site && cd my-laravel-site
+    ddev config --project-type=laravel --docroot=public
+
+    # Temporarily add the Laravel installer
```

Comment:
Since this is instructional, this may be better as a HEREDOC or something. I was going line by line and stumbled here. 

```suggestion
    # Temporarily add the Laravel installer as /usr/local/bin/laravel in the web container
```